themes/steeef.json

```diff
@@ -24,4 +24,8 @@
 	"hg": {
 		"prompt_dirty": "*"
 	}
+<<<<<<< HEAD
 }
+=======
+}
+>>>>>>> upstream/master
```

Comment:
fix this conflict

Reply:
Oh, apologies. I had an odd fork synchronisation issue with several of the theme files for some reason. Should be resolved.